_component_design/toggle.md

```diff
@@ -0,0 +1,181 @@
+helix-ui-css: true
+helix-ui-javascript: true
+pagelink: https://rackerlabs.github.io/helix-ui/components/toogle/
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a toggle when you want to provide the user control over how information is presented on a control panel page.
+
+For example, the user can use a toggle to switch the display of information from a card view that displays each entry as a tile on a grid, to a list view that displays each entry as an item in a list.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a toggle:
+
+- A toggle set must include a minimum of two view options.
+- Do not provide more than five view options in a toggle set.
+- The user can select only one view option at a time.
+- One view option must always be selected.
+- You can configure any of the view options as the default option.
+- Because a toggle does not pair the text with the icon, use tooltips to reveal what each option does.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Use the following specifications when you construct a toggle.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A toggle is composed of the following elements:
+
+- **Control container (required)**: The control container holds the option container and the option icon.
+- **Option container (required)**: The option container holds the option icon.
+- **Option icon (required)**: The option icon corresponds to how information displays on a control panel page. For example, the card view icon displays information on a set of tiles. The list view icon formats the information in a list.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Toggle composition"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/toggle/toggle_composition.png" width="262px"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Construct the control container so that it is 32px by 32px. Construct the option icon so that it is 16px by 16px.
```

Comment:
the control container wraps the WHOLE object. So it's 32 high X n-wide (depending on how many options are presented).

Reply:
ok, updated this wording.